include/zephyr/drivers/rtc/nxp_pcf85263a.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Fraunhofer AICOS
+ *
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_RTC_PCF8526A_H_
+#define ZEPHYR_INCLUDE_DRIVERS_RTC_PCF8526A_H_
+
+#include <time.h>
+
+#define PCF85263A_ALARM_FLAGS_USE_INTA           BIT(0)
+#define PCF85263A_ALARM_FLAGS_USE_INTB           BIT(1)
+
+typedef void (*nxp_pcf8523a_alarm_callback_t)(const struct device *dev,
+					 uint8_t chan_id, uint32_t ticks,
+					 void *user_data);
+
+struct nxp_pcf85263a_alarm_cfg {
+	time_t time;
+	nxp_pcf8523a_alarm_callback_t callback;
+	uint8_t flags;
+};
+
+int nxp_pcf85263a_get_value(const struct device *dev, uint64_t *value);
```

Comment:
what is the purpose of exposing those APIs?

Reply:
If is intended to be kept then those needs a proper documentation.

Reply:
My goal with this was to "isolate" the use of PCF85263A as a RTC from the use of it as a counter.
Or would it make more sense to hide all "counter" functions into its subsystem and provide only RTC extra functions publicly? At least [nxp_pcf85263a_set_value()](https://github.com/diocorreia/zephyr/blob/22f8c513e882925e9b792910eada9aadc2f75e2e/include/zephyr/drivers/rtc/nxp_pcf85263a.h#L25) should be public.

Reply:
Since the PCF85263A is designed to be used as an RTC, why not expose the device as an RTC using the [RTC API](https://docs.zephyrproject.org/latest/hardware/peripherals/rtc.html)? Apologies if I'm missing something here, but it seems this would allow for the time to be set without needing a custom API extension.